src/common/image-optimisation.ts

```diff
@@ -33,14 +34,16 @@ export const optimiseImage = async (
     };
   }
 
-  const desiredImageFormat = canAcceptAvif ? ImageTypes.avif : ImageTypes.webp;
+  logger.info({
+    message: `converting image to format: ${format}`,
+  });
 
-  pipe.toFormat(desiredImageFormat, {
+  pipe.toFormat(format as unknown as AvailableFormatInfo, {
```

Comment:
Can we have the type declaration in the parameters match what is required here please

Reply:
i cant for the life of me get it to convert neatly to their type for this

Reply:
yeah all good 👍